src/docs/sphinx/BuildingAscent.rst

```diff
@@ -595,7 +604,7 @@ Here is script that shows how to build additional dependencies for bflow-stats (
 .. _paraview_ascent_support:
 
 ParaView Support
-----------------
+~~~~~~~~~~~~~~~~~
```

Comment:
Missing this title as well
![no_paraview_title](https://github.com/user-attachments/assets/d35cb1ff-a26c-4140-9455-73f0b2f1fabd)